pages/location.jsx

```diff
@@ -75,8 +75,8 @@ var Location = React.createClass({
             <h1>How to get there</h1>
             <div className="illustration flip-it">
               <div className="illustration-image-container">
-                <ImageTag src1x="/assets/images/img-train.svg"
-                  height="219" width="214"
+                <ImageTag src1x="/assets/images/icon_train.svg"
+                  height="auto" width="260"
```

Comment:
Since the height property gets passed to an `img` tag as an attribute, I don't believe `auto` is valid. Same goes for the rest of these